ftl/core/help.ftl

```diff
@@ -0,0 +1,20 @@
+### Text shown in Help pages
+
+
+## Header
+
+# Used as title attribute for links to chapters in the manual
+help-open-manual-chapter = Open chapter { $name } in Anki manual
+
+## Body
+
+# Newly introduced settings may not have an explanation yet
+help-no-explanation =
+    Whoops! There doesn't seem to be an explanation for this setting yet.
+    
+    You can help us complete this help page on { $link }.
+
+## Footer
+
+# Link to more detailed information in the manual
+help-for-more-info = For more information, see { $link } in the manual.
```

Comment:
Can we cut out the other two if they're not used?

Reply:
I'm sorry, but I'm not sure what you mean with "other two". The only string that isn't used currently is the one that appears, when no `help` string is provided for a modal item. Technically all of them are used in the code.

Reply:
The first two don't seem to be used - what am I missing?

```
[dae] code/desktop/anki [main]  % gh pr checkout 2139
remote: Enumerating objects: 165, done.
remote: Counting objects: 100% (140/140), done.
remote: Compressing objects: 100% (49/49), done.
remote: Total 111 (delta 89), reused 77 (delta 62), pack-reused 0
Receiving objects: 100% (111/111), 21.91 KiB | 3.65 MiB/s, done.
Resolving deltas: 100% (89/89), completed with 20 local objects.
From github.com:ankitects/anki
   c7b51ef2a..9ae039b2b  refs/pull/2139/head -> deck-options-rework
Switched to branch 'deck-options-rework'
[dae] code/desktop/anki [deck-options-rework]  % rg openManual
zsh: exit 1     rg openManual
[dae] code/desktop/anki [deck-options-rework]  % rg open_manual
zsh: exit 1     rg open_manual
[dae] code/desktop/anki [deck-options-rework]  % rg noExplan
zsh: exit 1     rg noExplan
[dae] code/desktop/anki [deck-options-rework]  % rg no_explan
zsh: exit 1     rg no_explan
```

Reply:
https://github.com/kleinerpirat/anki/blob/9ae039b2b7a4178c00b16b355a04d81a6d509a4b/ts/deck-options/HelpModal.svelte#L72

![image](https://user-images.githubusercontent.com/62722460/197497129-d3c3f0e8-67a0-4bab-ad98-a1bf4b2f8ae1.png)


---
The other one: https://github.com/kleinerpirat/anki/blob/9ae039b2b7a4178c00b16b355a04d81a6d509a4b/ts/deck-options/HelpSection.svelte#L32

Reply:
🤦 Silly me on searching incorrectly.

I didn't notice the clickable headings before - they're not the most discoverable. What about having a "View in manual" link below the heading in smaller text instead of making the heading itself clickable? Alternatively we could just rely on the links in individual sections, and not have one to the chapter itself.